src/modules/service-catalog/ServiceCatalogList.tsx

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useEffect, useState } from "react";
+import ServiceCatalogListItem from "./components/service-catalog-list-item/ServiceCatalogListItem";
+import type { ServiceCatalogItem } from "./data-types/ServiceCatalogItem";
+import { Col, Row } from "@zendeskgarden/react-grid";
+import styled from "styled-components";
+import { useTranslation } from "react-i18next";
+
+import { CursorPagination } from "@zendeskgarden/react-pagination";
+
+const StyledCol = styled(Col)`
+  margin-top: ${(props) => props.theme.space.sm};
+  flex: 0 0 25%; /* Set width to 25% for 4 columns */
+  max-width: 25%; /* Set max-width to 25% for 4 columns */
+  padding: 0 24px 24px 0;
```

Comment:
Ok, I removed it and changed to Grid, from designs the space should be 24px but since default is 20px maybe I will wrote to Guntis that we will stay with that number.